src/server_manager/web_app/ui_components/outline-gcp-create-server-app.ts

```diff
@@ -337,12 +340,17 @@ export class GcpCreateServerApp extends LitElement {
   }
 
   private async refreshBillingAccounts(): Promise<void> {
-    this.billingAccounts = await this.account.listBillingAccounts();
+    try {
```

Comment:
We should call this function on "Next", catch the error, and display to the user.
However, no display to the user is needed in the refresh loop.
Move the try/catch to the setInterval call to support the different behavior.

Reply:
Done